apps/miner/lib/miner.ex

```diff
@@ -20,6 +20,8 @@ defmodule Miner do
       List.first(chain)
       |> Block.initialize
 
+	difficulty = block.difficulty
```

Comment:
difficulty should be its own variable, not based off of the difficulty of the block